tests/rpmsigdig.at

```diff
@@ -1988,6 +1988,97 @@ tagno: 267 (Dsa)
 [])
 RPMTEST_CLEANUP
 
+RPMTEST_SETUP_RW([multisign with disabled algorithms])
+AT_KEYWORDS([rpmsign signature multisig v4])
+RPMTEST_SKIP_IF([test x$PGP = xdummy])
+
+RPMTEST_CHECK([
+cat << EOF > ${HOME}/.config/rpm/macros
+%_openpgp_sign sq
+%_openpgp_sign_id 771B18D3D7BAA28734333C424344591E1964C5FC
+EOF
+
+runroot sq key import /data/keys/*.secret
+runroot rpmkeys --import /data/keys/*.pub
+
+cp "${RPMTEST}"/data/RPMS/hello-2.0-1.x86_64.rpm "${RPMTEST}"/tmp/
+runroot rpmsign --addsign --rpmv6 /tmp/hello-2.0-1.x86_64.rpm &> /dev/null
+runroot rpmsign --addsign --rpmv6 --key-id 152BB32FD9CA982797E835CFB0645AEC757BF69E /tmp/hello-2.0-1.x86_64.rpm &> /dev/null
+runroot rpmsign --addsign --rpmv6 --key-id E8A62C0512B06B5D2183BA207F1C21F95F65BBE8 /tmp/hello-2.0-1.x86_64.rpm &> /dev/null
+],
+[0],
+[ignore],
+[ignore])
+
+# RSA disabled
+RPMTEST_CHECK([
+runroot cp /etc/crypto-policies/back-ends/rpm-sequoia.config /tmp/
+runroot sed -i '/^rsa/s/always/never/g' /etc/crypto-policies/back-ends/rpm-sequoia.config
+runroot rpmkeys -Kv /tmp/hello-2.0-1.x86_64.rpm
+],
+[0],
+[/tmp/hello-2.0-1.x86_64.rpm:
+    Header OpenPGP V4 ECDSA/SHA512 signature, key fingerprint: e8a62c0512b06b5d2183ba207f1c21f95f65bbe8: OK
+    Header OpenPGP V4 EdDSA/SHA512 signature, key fingerprint: 152bb32fd9ca982797e835cfb0645aec757bf69e: OK
+    Header OpenPGP V4 RSA/SHA512 signature, key fingerprint: 771b18d3d7baa28734333c424344591e1964c5fc: NOTTRUSTED
+    Header SHA256 digest: OK
+    Payload SHA256 digest: OK
+],
+[ignore])
+
+# EDDSA disabled
+RPMTEST_CHECK([
+runroot cp -f /tmp/rpm-sequoia.config /etc/crypto-policies/back-ends/
+runroot sed -i '/^cv25519/s/always/never/g' /etc/crypto-policies/back-ends/rpm-sequoia.config
```

Comment:
I think this should be ed25519. The cv25519 is key exchange/derive mechanism used in encryption:
```suggestion
runroot sed -i '/^ed25519/s/always/never/g' /etc/crypto-policies/back-ends/rpm-sequoia.config
```

Reply:
So it smells like a bug someplace...

Reply:
This is all Greek to me, but https://gitlab.com/sequoia-pgp/sequoia/-/blob/main/openpgp/src/policy.rs?ref_type=heads#L1574 looks suspicious:

```
[...]
Curve::Ed25519 => Cv25519,
Curve::Cv25519 => Cv25519,
[...]
```

Reply:
Added a link to the ticket + the right action, rebased.